test/metabase/db/schema_migrations_test.clj

```diff
@@ -1156,3 +1156,29 @@
         (db.setup/migrate! db-type data-source :down 47)
         (testing "Rollback to the previous version should restore the column column, and set the default color value"
           (is (= "#31698A" (:color (t2/select-one :model/Collection :id collection-id)))))))))
+
+(deftest set-number-semantic-type-null-test
```

Comment:
I feel like this is an overkill.
- The thing it tests is quite trivial and it's not huge problem if it doesn't work.
- The thing that  is potentially dangerous (e.g., that it doesn't update a bunch of unrelated things) is not being tested.
- This is not a test we need to prevent regressions or specify some behavior.
- Migration tests are particularly slow.
- Manual tests (for h2, postgres, and mysql if you want to be very through) would be enough and possibly better.

Reply:
Thank you for the thoughtful feedback. I will mind that while writing migrations in future. As of now, test has been removed, as advised, and modified functionality tested manually for postgres, mysql and h2 as follows:

1. Start app.
2. Migrate down using `(dev/migrate! :down 47)`.
3. Set semantic_type for all rows having it null to 'type/Number'
4. Migrate up using `(dev/migrate! :up 48)`
5. Check whether 'type/Number' fields are reset to null.